cpp/include/cudf/table/experimental/row_operators.cuh

```diff
@@ -932,11 +965,14 @@ class self_comparator {
    *
    * `F(i,j)` returns true if and only if row `i` compares equal to row `j`.
    *
-   * @tparam Nullate A cudf::nullate type describing whether to check for nulls.
+   * @tparam Nullate A cudf::nullate type describing whether to check for nulls
```

Comment:
Well OK now that I'm seeing `tparam Nullate` everywhere in this PR I'm definitely more interested in resolving my previous comment about finding a way to standardize it if possible. Maybe we store a few standard docstrings in some central file and then use `copydoc`? It would be even more awesome if there was some way to have doxygen "detect" that a function has a Nullate template and automatically add in the parameter, but I don't even know if that's possible, and even if it is that might require way too much specialized scripting. Just thought I'd raise the possibility, please let me know if you think any of these are terrible ideas for any reason :) The automatic documentation might be too much magic in any case since devs won't know to expect that to work. A copydoc seems like a good balance of standardization without magic.

Reply:
`@copydetails` also would work, but it will create 2 sections in generated documentation for params.
doxygen aliases is another trick. We can define our own aliases in Doxyfile and use it anywhere in our code.

Reply:
That's a good counterpoint. Consistency in the rendered docs may be less important than visibility in the source for developers.